tools/koji-compose.py

```diff
@@ -56,40 +56,41 @@ def compose_request(distro, koji, arch):
 
 def main(distro, arch):
     cr = compose_request(distro, "https://localhost:4343/kojihub", arch)
-    print(json.dumps(cr))
+    print(json.dumps(cr), file=sys.stderr)
 
     r = requests.post("https://localhost/api/image-builder-composer/v2/compose", json=cr,
                       cert=("/etc/osbuild-composer/worker-crt.pem", "/etc/osbuild-composer/worker-key.pem"),
                       verify="/etc/osbuild-composer/ca-crt.pem")
     if r.status_code != 201:
-        print("Failed to create compose")
-        print(r.text)
+        print("Failed to create compose", file=sys.stderr)
+        print(r.text, file=sys.stderr)
         sys.exit(1)
 
-    print(r.text)
+    print(r.text, file=sys.stderr)
     compose_id = r.json()["id"]
+    print(compose_id)
```

Comment:
Alternatively could have written it to a file and then get it from there, but I think it is fine.

Reply:
I agree. A matter of personal preference I guess 🙂